web/src/app/services/crossword-data.service.ts

```diff
@@ -60,23 +60,36 @@ export class CrosswordDataService {
   }
 
   public getGame(gameId: string): Observable<Game> {
-    if (this.games[gameId]) {
-      return Observable.of(this.games[gameId]);
-    }
-
     return this.http.get(`${AppConfig.GAME_URL}/${gameId}`)
       .mergeMap((res: GameResponse) => {
         return this.http.get(`${AppConfig.GAME_URL}/${gameId}/clues`)
           .map((clueRes: CluesResponse) => this.extractClues(clueRes))
           .map((clues: Clue[]) => {
-            return new Game({
+            const game = new Game({
               id: gameId,
               crossword: this.extractCrossword(res, clues),
             });
+            game.deserializeBoard(res.board_state);
```

Comment:
We probably need to add an optional Game parameter (boardState) to the ctor.

Reply:
don't need to do this now

Reply:
agree